pkg/build/nodeimage/const_cni.go

```diff
@@ -20,7 +20,7 @@ package nodeimage
 The default CNI manifest and images are our own tiny kindnet
 */
 
-var defaultCNIImages = []string{"kindest/kindnetd:v20210326-1e038dc5"}
+var defaultCNIImages = []string{"docker.io/kindest/kindnetd:v20210326-1e038dc5"}
```

Comment:
should we use a constant?
 in case we need/have to switch to other registry that will make it easier

Reply:
ups, it is embedded into the manifests, forget it

Reply:
this is just the fully qualified value, it's already implicitly `docker.io/` for images that do not specify a host.
except on podman, which is often configured to try a redhat registry first IIRC. but kubernetes etc. will resolve this to `docker.io/kindest/kindnetd`.